### PR TITLE
Add codebase

### PR DESCRIPTION
**Pull request by teckim**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (teckim) [inline css/style.css:1]: - [ ] Please remove the empty line at the beginning of the file, this makes your code not clean.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      position: fixed;
+      top: 0;
+    }
+
+    @media (max-width: 600px) {
+      #header-img {
+        width: 150px;
+      }
+    }
+  </style>
```
Comment (teckim) [inline index.html:23]: - [ ] I think moving the styling to a dedicated file, will make the code cleaner.

**Review by teckim — COMMENTED** [review]
# Hello @teckim 👋 

Great progress on the project, your design looks absolutely amazing 🔥 🔥   
There are some issues that you still need to work on but you are almost there! 🌞 

## Required Changes ♻️
- [ ] Kindly make sure to add a descriptive Pull request summary.
- [ ] Please add the Microverse linter configurations [read more](https://github.com/microverseinc/linters-config).
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

File: css/style.css (hunk 1 above)
Comment (phelian23) [reply css/style.css:1]: - [ ]  Nicely done 👍

File: index.html (hunk 2 above)
Comment (phelian23) [reply index.html:23]: - [ ]  Nicely done 👍

**Review by phelian23 — APPROVED** [review]
Hi @teckim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by teckim** [comment]
> Hi @teckim,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @phelian23, thank you for your response.
sorry, but I still see submit status on the activity, is it normal?

**Pull request merged**